x-pack/plugins/upgrade_assistant/server/lib/es_deprecations_status.ts

```diff
@@ -18,10 +18,24 @@ import {
 export function getShardCapacityDeprecationInfo({
   symptom,
   details,
+  status,
 }: {
   details: any;
   symptom: any;
+  status: estypes.HealthReportIndicatorHealthStatus;
 }) {
+  // When we dont have a details field for our indicator, we can only report
+  // the symptom to the user given that's the only information about the deprecation
+  // we have.
+  if (status === 'unknown' && !details) {
```

Comment:
Why do we need to also check if `status === 'unknown'`?

I wonder if it wouldn't be better to just check the `details` field - this would future-proof against scenarios where there are no details provided and `status` is not `unknown`.

Reply:
Yeah good point! I've addressed that with https://github.com/elastic/kibana/pull/187326/commits/0ee14ca8469090bcd19c012db572a832f4385d05